linera-ethereum/src/test_utils/mod.rs

```diff
@@ -121,12 +124,18 @@ impl SimpleTokenContractFunction {
         // 2: gettting the balance transaction stuff
```

Comment:
why the numbers? (especially the one in `new` is weird)

Reply:
Also: the number of `t`s is off. :smile:

Reply:
Ok, remove those.